vendor/vendor.json

```diff
@@ -11,20 +11,20 @@
 			"versionExact": "0.1.0"
 		},
 		{
-			"checksumSHA1": "yceqWuIbFzntmSHl66OwNoP/c6E=",
+			"checksumSHA1": "GrgODPqXv7QebQjfjrxMHctK0wE=",
 			"path": "github.com/Financial-Times/message-queue-go-producer/producer",
-			"revision": "163efbf93f1a2ba839b317d26e37426bcfa3850a",
-			"revisionTime": "2017-03-20T10:55:12Z",
-			"version": "v0.1.0",
-			"versionExact": "v0.1.0"
+			"revision": "e0d20e2c97fff205138d33c3dee9347878640b5e",
+			"revisionTime": "2017-05-26T06:27:13Z",
+			"version": "0.2.0-xp-remove-kafka-topic-check-rc.1",
+			"versionExact": "0.2.0-xp-remove-kafka-topic-check-rc.1"
 		},
 		{
-			"checksumSHA1": "/kMYtkcxWV4dZNvJaEGkDUwt3TA=",
+			"checksumSHA1": "mirNgwWRdRte0ku/g4u4fNbzQP0=",
 			"path": "github.com/Financial-Times/message-queue-gonsumer/consumer",
-			"revision": "4e173298922ba4c10721b1b5b3e0d35be7f162e9",
-			"revisionTime": "2017-03-20T10:34:32Z",
-			"version": "0.3.0",
-			"versionExact": "0.3.0"
+			"revision": "0a481ff94df47f766a74867475247eaf1ed40107",
+			"revisionTime": "2017-05-26T12:08:30Z",
+			"version": "0.4.0-xp-remove-kafka-topic-check-rc.1",
+			"versionExact": "0.4.0-xp-remove-kafka-topic-check-rc.1"
```

Comment:
as discussed, use the new stable version for consumer and producer after they are tested